az-release-pipelines/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$AZURE_PIPELINE_ORGANIZATION" ]; then
+    echo "\$AZURE_PIPELINE_ORGANIZATION is not set."
+    exit 1
+fi
+
+if [ -z "$AZURE_PIPELINE_PROJECT" ]; then
+    echo "\$AZURE_PIPELINE_PROJECT is not set."
+    exit 1
+fi
+
+if [ -z "$AZURE_PIPELINE_TOKEN" ]; then
+    echo "\$AZURE_PIPELINE_TOKEN is not set."
+    exit 1
+fi
+
+if [ -z "$AZURE_PIPELINE_NAME" ]; then
+    echo "\$AZURE_PIPELINE_NAME is not set."
+    exit 1
+fi
+
+    
+AZDEVOPS_URL="https://dev.azure.com/${AZURE_PIPELINE_ORGANIZATION}/"
+vsts configure --defaults instance=${AZDEVOPS_URL} project=${AZURE_PIPELINE_PROJECT}
+    
+vsts login --token ${AZURE_PIPELINE_TOKEN}
+    
+echo "Triggering Azure release pipeline: ${AZURE_PIPELINE_NAME}"
+vsts release create --definition-name ${AZURE_PIPELINE_NAME}
```

Comment:
where are you taking the build numbers?

Reply:
As per the spec, Build number is not present. I have added the support in vsts-cli to take pair of alias and build number.